partitioning_raid-fat_format-selected-20190125.json

```diff
@@ -10,6 +10,6 @@
   ],
   "properties": [],
   "tags": [
-    "partitioning_raid-fat_format-selected"
+    "partitioning_fat-format-selected"

```

Comment:
I think we should be consistent with name of the files and needle tag, wdyt? I see you delete the raid part so maybe worth to do the same in the name of the file, .png and .json, but we don't care too much about names, so up to you, it is not like you have to do it :)